### PR TITLE
#64: Add documentation for paginated results (closes #64)

### LINKED ISSUE #64: Add documentation for paginated results
[Trello card](https://trello.com/c/58c018cfe6458f6fbd933209/)

The `Paginated` object introduced in https://github.com/contactlab/contacthub-sdk-nodejs/pull/62/files should be documented in the README, where `getCustomers()` and `getEvents()` are documented.

### PR DESCRIPTION
**Pull request by ascariandrea**
Closes #64

### tests performed
Links works fine!

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.}

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -487,6 +487,22 @@ version of the `Customer`.
 ch.removeLike(customerId, likeId)
 ```
 
+## Paginated Result
+The `PaginatedResult` helps to browse api through pagination.
```
Comment (dhinus) [inline README.md]: I would rephrase this as
> A `PaginatedResult` object is returned whenever the API returns a paginated list of elements.

File: README.md [hunk 2]
```diff
@@ -487,6 +487,22 @@ version of the `Customer`.
...
+The `page.prev` and `page.next` are utlity methods to fetch the adjacent pages.
```
Comment (dhinus) [inline README.md]: `s/utlity/utility/`

Comment (ascariandrea) [inline README.md]: I've amended the commit cause it seemed overkill to add a new one 🙃
